build_tools/ci/cpu_comparison/run.py

```diff
@@ -708,6 +714,22 @@ def _execute(self, config):
         input_args = generate_inputs(
             test_file_D, config.get_test_dir(test_name_D), seed=1
         )
+
+        if self.run_benchmark:
+            print(f"Performance benchmark: {test_file_D}")
+            benchmark_aie_kernel_time(
+                config,
+                aie_vmfb_C,
+                input_args,
+                self.function_name,
+                test_name_C,
+                self.n_repeats,
+                self.n_kernel_runs,
+                self.n_reconfigure_runs,
+                time_unit="ms",
```

Comment:
I suggest to use "us" to keep the same unit as the other tests. Otherwise you need to change the unit when plotting this result.